TaskSharper.DataAccessLayer.Google/Authentication/GoogleAuthentication.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Util.Store;
+
+namespace TaskSharper.DataAccessLayer.Google.Authentication
+{
+    public static class GoogleAuthentication
+    {
+        private static readonly string[] Scopes = { CalendarService.Scope.Calendar };
+        public static UserCredential Authenticate()
+        {
+            UserCredential credential;
+
+            using (var stream = new FileStream("client_secret.json", FileMode.Open, FileAccess.Read))
+            {
+                string credPath = Environment.GetFolderPath(Environment.SpecialFolder.Personal);
```

Comment:
This i fine for now, but at some point I think we have to consider an appropriate location for the client information to reside.

Reply:
Agreed.